api/graphql/opening_hours/opening_hours_types.py

```diff
@@ -1,4 +1,7 @@
+import datetime
+
 import graphene
+import pytz
```

Comment:
I don't know if there is a lot we can do here, but I believe this is a good thing to be aware of.

Django will deprecate pytz in Django [v4 and remove support in v5](https://docs.djangoproject.com/en/4.0/releases/4.0/#zoneinfo-default-timezone-implementation), but we are still using python v3.8. The new standard will be available in python 3.9. So, we cannot start using the new [zoneinfo](https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo) yet.